### PR TITLE
Competition controller + fixed eslint errors + create game page

### PR DESCRIPTION
**Pull request by D1LL1G4F**

### CODE REVIEW

File: GamingManagement-view/package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
   },
   "scripts": {
     "types": "tsc --noEmit",
-    "lint": "eslint . --max-warnings 0",
+    "lint": "eslint . --max-warnings 0src/components/Team/index.tsx",
```
Comment (oreqizer) [inline GamingManagement-view/package.json:27]: toto vymaz pls

File: GamingManagement-view/src/services/useState.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import * as React from "react";
 
-type State<E extends Error> = null | { error: E | null };
+type State<E extends Error> = null | { error: E | null, created?: boolean };
```
Comment (oreqizer) [inline GamingManagement-view/src/services/useState.ts:3]: toto daj prec pls

File: GamingManagement-view/package.json (hunk 1 above)
Comment (D1LL1G4F) [reply GamingManagement-view/package.json]: ✅  done

File: GamingManagement-view/src/services/useState.ts (hunk 2 above)
Comment (D1LL1G4F) [reply GamingManagement-view/src/services/useState.ts]: nicomu to nevadilo ale je to prec

**Pull request merged**